### PR TITLE
Bug fixes for issues in code-migration

### PR DESCRIPTION
**Pull request by InvincibleZeal**

### CODE REVIEW

File: src/languages/en.json [hunk 1]
```diff
@@ -24,15 +24,16 @@
     "invoice.amount": "Amount",
     "invoice.notes": "Notes",
     "invoice.date": "Date",
-    "invoice.cutomse.name": "Customer",
+    "invoice.customer.name": "Customer",
     "invoice.paid.status": "Paid Status",
     "invoice.amount.due": "Amount Due",
     "invoice.change": "change",
-    "invoice.add.on.item": "Add an item",
+    "invoice.add.an.item": "Add an item",
```
Comment (rparsh) [inline src/languages/en.json:31]: you `.` to divide section. here it should be `invoice.addAnItem`

File: src/languages/en.json [hunk 2]
```diff
@@ -24,15 +24,16 @@
...
     "invoice.total.amount": "Total Amount",
     "invoice.model": "Model",
     "no.records": "There are no records available",
-    "invoice.add.customer.details": "Add  Customer details",
+    "invoice.select.customer": "Select Customer",
```
Comment (rparsh) [inline src/languages/en.json:35]: same here fix everywhere invoice.selectCustomer

File: src/notification/useNotification.jsx [hunk 3]
```diff
@@ -1,6 +1,22 @@
 import { useContext } from "react";
 import { NotificationContext } from "./NotificationContext";
 
+/**
```
Comment (rparsh) [inline src/notification/useNotification.jsx:4]: write Readme.md copy from issue(https://github.com/InvincibleZeal/billbook/issues/57) and improve it.

File: src/languages/en.json (hunk 2 above)
Comment (gautism1) [reply src/languages/en.json:35]: will do the changes in all possible keys

Comment (rparsh) [reply src/languages/en.json:35]: @gautism1 please do fixes in this branch

**Review by smilegupta — APPROVED** [review]

**Pull request merged**